addon/manifests/chart/templates/service.yaml

```diff
@@ -0,0 +1,26 @@
+---
+{{- if .Values.prometheus.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "controller.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "controller.fullname" . }}
+    chart: {{ template "search.chart" . }}
```

Comment:
Maybe this should be `controller.chart` for consistency. Also need to update in `_helpers.tpl`